Lib/test/test_pprint.py

```diff
@@ -327,6 +327,12 @@ def test_nested_indentations(self):
         'third': 3}]"""
         self.assertEqual(pprint.pformat(o, indent=4, width=41), expected)
 
+        expected = "[   [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],\n" \
```

Comment:
I think you can write a helper function, something like `assert_pprint(self, obj, expected, indent=1, width=80, depth=None)` (mirroring the defaults of `pprint.pprint`), that then tests both `pprint.pformat` to a string and `pprint,pprint` to a stream. Then use it everywhere instead of duplicating the expected values everywhere. You'll probably need to ignore the trailing newline on the stream version.